aqt/archives.py

```diff
@@ -155,44 +160,69 @@ def __init__(
             self.logger = getLogger("aqt")
         self.archives = []
         self.mod_list = []
-        qt_ver_num = self.version.replace(".", "")
-        self.qt_ver_base = self.version[0:1]
         if all_extra:
             self.all_extra = True
         else:
             for m in modules if modules is not None else []:
                 self.mod_list.append(
-                    "qt.qt{}.{}.{}.{}".format(self.qt_ver_base, qt_ver_num, m, arch)
+                    "qt.qt{0}.{1}{2}{3}.{4}.{5}".format(
+                        self.version.major,
+                        self.version.major,
+                        self.version.minor,
+                        self.version.patch,
+                        m,
+                        arch,
+                    )
```

Comment:
There are a couple of these that could be a little less redundant. You could use:
```python
 "qt.qt{0}.{0}{1}{2}.{3}.{4}".format(
    self.version.major,
    self.version.minor,
    self.version.patch,
    m,
    arch,
)
```
...but that's a stylistic choice, and probably not very important.